src/types.rs

```diff
@@ -1,15 +1,5 @@
-use ruby_sys::types;
+pub use ruby_sys::types::*;
```

Comment:
😅 

This file was identical to `ruby-sys`'s `types.rs` + `Callback` type. So I decided just to reexport original `types.rs` here